app/routes/users/components/UserProfile.js

```diff
@@ -135,12 +136,21 @@ export default class UserProfile extends Component<Props, UpcomingEventsProps> {
 
   renderFields() {
     const { user } = this.props;
-    const fields = Object.keys(fieldTranslations).filter(field => user[field]);
+    let fields = Object.keys(fieldTranslations).filter(field => user[field]);
     const tags = fields.map(field => {
       const translation = capitalize(fieldTranslations[field]);
```

Comment:
Haha, why didn't they just capitalize the strings themselves? :label: :laughing: